manifests/install.pp

```diff
@@ -40,10 +40,16 @@
     }
   }
 
-  $file = "atlassian-${jira::product}-${jira::version}.${jira::format}"
+  # https://www.atlassian.com/software/jira/downloads/binary/atlassian-jira-software-7.0.0-jira-7.0.0-x64.bin
+
+  if ($jira::version[0] == '7' ){
+    $file = "atlassian-jira-software-7.0.0-${jira::product}-${jira::version}${jira::format}"
+  }else {
+    $file = "atlassian-${jira::product}-${jira::version}${jira::format}"
```

Comment:
I usualy use selectors when there are more then 2 options ... but thinking on the future yes this could be usefull.